webpack.config.js

```diff
@@ -57,7 +49,7 @@ module.exports = {
     ]
   },
   resolve: {
-    root: path.resolve(__dirname, 'src'),
+    root: path.resolve(__dirname, ''),
```

Comment:
Should `resolve.root` be deleted instead? (or set to an empty array) AFAICT, we now want "module names" to only be resolved as Node modules.